ups/display_ginga.table

```diff
@@ -1,2 +1,3 @@
 envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
+setupRequired(base)
```

Comment:
Given that you use in the interface defined in afw, why wouldn't you `setupRequired(afw)`?

Reply:
Right.  I was trying to setup the minimum to build, but you are right that I should set up the packages needed to run.